### PR TITLE
[infra/runtime] Update TRIXEngine check

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates TRIXEngine version and header check to support upper 2.6.0 version. 
It includes cmake status message update.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Review by ragmani — DISMISSED** [review]
LGTM

**Comment by glistening** [comment]
[2023-09-26T06:39:39.879Z] [  PASSED  ] 10 tests.

[2023-09-26T06:39:39.879Z] ============================================

[2023-09-26T06:39:39.879Z] Finishing set 8: /tmp/tmp.5gKay6hfFv/Product/out/unittest/test_cker...

[2023-09-26T06:39:39.879Z] ============================================

[2023-09-26T06:39:39.879Z] ============================================

[2023-09-26T06:39:39.880Z] Completed total 8 set of unittest

[2023-09-26T06:39:39.880Z] Unittest end

[2023-09-26T06:39:39.880Z] ============================================

[2023-09-26T06:39:39.880Z] Test NPUD

script returned exit code 1

File: infra/nnfw/cmake/packages/TRIXEngineConfig.cmake [hunk 1]
```diff
@@ -31,12 +42,14 @@ function(_TRIXEngine_import)
     if(NOT TARGET trix_engine)
       add_library(trix_engine INTERFACE)
       target_link_libraries(trix_engine INTERFACE ${TRIXEngine_LIB})
-      target_include_directories(trix_engine INTERFACE ${TRIXEngine_INCLUDE_DIR})
-    endif(NOT TARGET trix_engine)
+      target_include_directories(trix_engine INTERFACE ${TRIXEngine_INCLUDE_DIR} ${NPUBINFMT_INCLUDE_DIR})
+     endif(NOT TARGET trix_engine)
```
Comment (glistening) [inline infra/nnfw/cmake/packages/TRIXEngineConfig.cmake:46]:
```suggestion
    endif(NOT TARGET trix_engine)
```

File: infra/nnfw/cmake/packages/TRIXEngineConfig.cmake [hunk 2]
```diff
@@ -31,12 +42,14 @@ function(_TRIXEngine_import)
...
   endif(NOT TRIXEngine_FOUND)
 
   set(TRIXEngine_FOUND ${TRIXEngine_FOUND} PARENT_SCOPE)
-  set(TRIXEngine_INCLUDE_DIRS ${TRIXEngine_INCLUDE_DIR} PARENT_SCOPE)
+  set(TRIXEngine_INCLUDE_DIRS ${TRIXEngine_INCLUDE_DIR} ${NPUBINFMT_INCLUDE_DIR} PARENT_SCOPE)
+
+
 endfunction(_TRIXEngine_import)
```
Comment (hseok-oh) [inline infra/nnfw/cmake/packages/TRIXEngineConfig.cmake:53]:
```suggestion
endfunction(_TRIXEngine_import)
```

**Review by ragmani — APPROVED** [review]

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**